flows.yaml

```diff
@@ -1,4 +1,124 @@
 integrations:
+    airtable:
+        syncs:
+            tables:
+                runs: every day
+                description: >
+                    Lists all tables with their schema for all bases with a reference to
+                    the base id that
+
+                    the table belongs to
```

Comment:
new lines

Reply:
This is the automatically generated file so don't have much control over it + I don't think newlines cause issues when rendering in the UI.